core/market/decentralized/src/testing/mock_net.rs

```diff
@@ -131,71 +131,79 @@ impl MockNetInner {
-                    topic,
-                    addr,
-                    node_id
-                );
-                let caller = caller.clone();
-                let msg = msg.clone();
-                Arbiter::spawn(async move {
-                    let _ = local_bus::send(addr.as_ref(), &caller, msg.as_ref()).await;
-                });
-            }
-            async move { Ok(Vec::from(resp.as_ref())) }
-        });
-
-        local_bus::subscribe(FROM_BUS_ID, move |caller: &str, addr: &str, msg: &[u8]| {
-            let mock_net = MockNet::default();
-            let data = Vec::from(msg);
-            let caller = caller.to_string();
-            let addr = addr.to_string();
-
-            async move {
-                let (from, local_addr) = mock_net
-                    .translate_address(addr)
-                    .await
-                    .map_err(|e| Error::GsbBadRequest(e.to_string()))?;
-
-                log::debug!(
-                    "[MockNet] Sending message from [{}], to address [{}].",
-                    &caller,
-                    &local_addr
-                );
-                Ok(local_bus::send(&local_addr, &from.to_string(), &data).await?)
-            }
-        });
+                                topic,
+                                addr
+                            );
+                            continue;
+                        }
+                    };
+
+                    log::debug!(
+                        "BCasting on {} to address: {}, node: [{}]",
+                        topic,
+                        addr,
+                        node_id
+                    );
+                    let caller = caller.clone();
+                    let msg = msg.clone();
+                    Arbiter::spawn(async move {
+                        let _ = local_bus::send(addr.as_ref(), &caller, msg.as_ref()).await;
+                    });
+                }
+                async move { Ok(Vec::from(resp.as_ref())) }
+            },
+            (),
+        );
+
+        local_bus::subscribe(
+            FROM_BUS_ID,
+            move |caller: &str, addr: &str, msg: &[u8]| {
+                let mock_net = MockNet::default();
+                let data = Vec::from(msg);
+                let caller = caller.to_string();
+                let addr = addr.to_string();
+
+                async move {
+                    let (from, local_addr) = mock_net
+                        .translate_address(addr)
+                        .await
+                        .map_err(|e| Error::GsbBadRequest(e.to_string()))?;
+
+                    log::debug!(
+                        "[MockNet] Sending message from [{}], to address [{}].",
+                        &caller,
+                        &local_addr
+                    );
+                    Ok(local_bus::send(&local_addr, &from.to_string(), &data).await?)
+                }
+            },
+            (),
+        );
```

Comment:
Is it only formatting change? Am I right??

Reply:
@nieznanysprawiciel `subscribe` now takes 2 arguments (the second one being `()` in this case)